pytket/docs/changelog.rst

```diff
@@ -1,9 +1,10 @@
 Changelog
 =========
 
-(unreleased)
-------------
+1.31.0 (August 2024)
+--------------------
 
+* remove types-pkg_resources dependency
```

Comment:
GIven that the removal of this package from pypi renders pytket uninstallable I think we should announce this more prominently in the docs. What do you think?

Reply:
I can do this in a separate PR

Reply:
Yes we should do that. Feel free to make a second PR for that.